### PR TITLE
tmg - bs345

### PR DESCRIPTION
**Pull request by Polkas**
linked to https://github.com/insightsengineering/teal/issues/719

- [x] fixed the missing_data module layout for bootstrap 4 and 5.
- [x] fixed grouping of the panel_item in the distribution module.

### CODE REVIEW

File: R/tm_missing_data.R [hunk 1]
```diff
@@ -123,7 +123,15 @@ ui_page_missing_data <- function(id, datasets, pre_output = NULL, post_output =
             datanames,
             function(x) {
               conditionalPanel(
-                sprintf("$(\"#%s > li.active\").text().trim() == \"%s\"", ns("dataname_tab"), x),
+                # supporting the bs3 and higher version at the same time
+                sprintf(
+                  paste0(
+                    "$(\"#%1$s > li.active\").text().trim() == '%2$s' || ",
+                    "$(\"#%1$s > li a.active\").text().trim() == '%2$s'"
+                  ),
```
Comment (nikolas-burkoff) [inline R/tm_missing_data.R:131]: I wonder if we can refactor this out into a function as it is used in a few different places

Comment (Polkas) [reply R/tm_missing_data.R]: is_tab_active_js was created

File: R/tm_g_distribution.R [hunk 2]
```diff
@@ -257,30 +257,30 @@ ui_distribution <- function(id, ...) {
...
+                icon("circle-info"),
+                span(
+                  class = "tooltiptext",
+                  "Default parameters are optimized with MASS::fitdistr function."
+                )
               )
-            )
+            ),
+            choices = c("normal", "lognormal", "gamma", "unif"),
+            selected = NULL, multiple = FALSE
```
Comment (Polkas) [inline R/tm_g_distribution.R:277]:
```suggestion
            selected = NULL, 
            multiple = FALSE
```

File: R/utils.R [hunk 3]
```diff
@@ -289,3 +289,16 @@ include_css_files <- function(pattern = "*") {
   }
   return(shiny::singleton(shiny::tags$head(lapply(css_files, shiny::includeCSS))))
 }
+
```
Comment (Polkas) [inline R/utils.R:292]:
```suggestion


```

**Comment by asbates** [comment]
LGTM

**Comment by donyunardi** [comment]
Found this issue while testing `tm_missing_data`:

image

To reproduce:
```
library(scda)
library(teal.modules.general)
library(nestcolor)

ADSL <- synthetic_cdisc_data("latest")$adsl
ADRS <- synthetic_cdisc_data("latest")$adrs

app <- init(
  data = cdisc_data(
    cdisc_dataset("ADSL", ADSL, code = "ADSL <- synthetic_cdisc_data(\"latest\")$adsl"),
    cdisc_dataset("ADRS", ADRS, code = "ADRS <- synthetic_cdisc_data(\"latest\")$adrs"),
    check = TRUE
  ),
  modules = modules(
    tm_missing_data(
      ggplot2_args = list(
        "Combinations Hist" = teal.widgets::ggplot2_args(
          labs = list(subtitle = "Plot produced by Missing Data Module", caption = NULL)
        ),
        "Combinations Main" = teal.widgets::ggplot2_args(labs = list(title = NULL))
      )
    )
  )
)
if (TRUE) {
  shinyApp(app$ui, app$server)
}
```

And once the app run, click the “Select only vars with missing” box on the encoding panel.

I switched to main branch and I ran into the same behavior so it's possible that this is not related to our bootstraps PR.

**Review by donyunardi — APPROVED** [review]
Thanks for catching this @Polkas:
https://github.com/insightsengineering/teal.modules.general/commit/8b55bef981fe1cdc264edb14ab4b4636113cbfe9

tested couple modules including distribution and missing_data.
All features seems to be functioning correctly.
Approved.

**Pull request merged**